### PR TITLE
Nonwiz patch 1

### PR DESCRIPTION
**Pull request by nonwiz**
Adding my personal site and blog to showcase.

### CODE REVIEW

**Comment by nonwiz** [comment]
Added my fourth iteration with lume to showcase.

File: showcase.yml [hunk 1]
```diff
@@ -302,3 +302,10 @@ sites:
     url: https://dominick.cc/
     description: The personal site of Dominick Peluso.
     code: https://github.com/dominickp/dom-deno
+
+  - title: Nonwiz's Homepage
+    img: /img/showcase/nonwiz_compressed.png
```
Comment (oscarotero) [inline showcase.yml:307]: The extension of the image must be .webp, as Lume convert automatically all images to webp.

**Review by oscarotero — COMMENTED** [review]
Thanks for your pull request!

Just a couple of things:

- Can you move the site to the 2nd position (after lume.land site)? I'm adding new entries to the top, so they are more visible.
- The image is not correctly loaded. I left you a comment about that.

**Comment by nonwiz** [comment]
> Thanks for your pull request!
> 
> Just a couple of things:
> 
>     * Can you move the site to the 2nd position (after lume.land site)? I'm adding new entries to the top, so they are more visible.
> 
>     * The image is not correctly loaded. I left you a comment about that.

Somehow when I run locally, the screenshot also is not being rendered although I already converted them into webp format. Is there anything I missed?

**Comment by oscarotero** [comment]
You need to change the extension only in the yaml file, but keeping the png file in the source folder. Lume converts automatically png files to webp.

**Comment by oscarotero** [comment]
Thanks!!

**Pull request merged**